malApp.R

```diff
@@ -15,10 +15,25 @@ min_date<- substr(min_date[1,"min"], 1, 10)
 max_date <- read.csv("/home/mkhuphuli/hello/credentials.csv", header=TRUE) %>%
   get_data_fromDB(get_max_date)
 max_date<- substr(max_date[1,"max"], 1, 10)
-
+ 
 #get some general malaria data 
+'date_column <- "date_reported"
 df1 <- get_data_fromDB(credentials=credentials, get_malaria_Data) %>%
-  reported_case_counts()
+  reported_case_counts() %>% 
+  #add seasons and month according to date on which malaria case was reported
+  get_seasons(date_column=date_column)'
+
+df1 <-  read.csv("df1.csv", header=TRUE)
```

Comment:
df1 is not really a helpful var name, give it a name that lets anyone know what is does